docs/changelog.rst

```diff
@@ -0,0 +1,162 @@
+=========
+Changelog
```

Comment:
Should this file be linked somewhere in the docs?

Reply:
changed as discussed

Reply:
Did you push? I still see the old file